include/gatekeeper_gk.h

```diff
@@ -61,9 +126,30 @@ struct gk_config {
 
 	struct gk_instance *instances;
 	struct net_config  *net;
+	/*
+	 * XXX The LPM table used by the gk instances.
+	 * Here, we assume that all the gk instances are
```

Comment:
You can drop the tag `XXX` in this comment since no action is required.

I'd drop "Here" and start with the "We assume...".